lib/project_types/node/messages/messages.rb

```diff
@@ -83,52 +83,18 @@ module Messages
 
         generate: {
           help: <<~HELP,
-          Generate code in your Node project. Supports generating new billing API calls, new pages, or new webhooks.
-            Usage: {{command:%s generate [ billing | page | webhook ]}}
-          HELP
-          extended_help: <<~EXAMPLES,
-          {{bold:Examples:}}
-            {{cyan:%s generate webhook PRODUCTS_CREATE}}
-              Generate and register a new webhook that will be called every time a new product is created on your store.
-          EXAMPLES
+          {{red:The {{command:generate}} command will no longer be supported going forward, because the new Shopify libraries will provide these features instead.}}
 
-          error: {
-            name_exists: "%s already exists!",
-            generic: "Error generating %s",
-          },
+          Here are some resources to help you add the previously supported features to your app:
```

Comment:
Here and in the following suggestions, just trying to use more action-oriented language:

```suggestion
          You can complete any tasks previously supported by {{command:generate}} with these guides:
```